charmhelpers/core/hookenv.py

```diff
@@ -975,6 +986,22 @@ def action_get(key=None):
     return action_data
 
 
+@cached
+def function_get(key=None):
+    """Gets the value of an action parameter, or all key/value param pairs"""
+    # Fallback for older charms.
+    if not cmd_exists('function-get'):
+        return action_get(key)
```

Comment:
Hrm, this will lead to deprecation warnings being emitted even if the charm is using the correct code.  Seems like it would be better to just change the value of `cmd`.

Reply:
fixed